raml-headers-form.html

```diff
@@ -197,7 +223,7 @@
                     <label>Header name</label>
                     <input
                       is="anypoint-input"
-                      class="name-change-input"
+                      class="name-change-input anypoint-text-input"
```

Comment:
This is not anypoint branch. You should use the anypoint branch to have anypoint styling.